lib/Bindings/Python/OMModule.cpp

```diff
@@ -345,11 +384,13 @@ void circt::python::populateDialectOMSubmodule(py::module &m) {
   // Add the List class definition.
   py::class_<List>(m, "List")
       .def(py::init<List>(), py::arg("list"))
+      .def("getLoc", &List::getLoc, "Get location info")
       .def("__getitem__", &List::getElement)
       .def("__len__", &List::getNumElements);
 
   py::class_<Tuple>(m, "Tuple")
       .def(py::init<Tuple>(), py::arg("tuple"))
+      .def("getLoc", &Tuple::getLoc, "Get location info")
```

Comment:
The convention is to name Python methods snakecase, so these should probably be `get_loc`